src/components/NFText/NFText.tsx

```diff
@@ -0,0 +1,108 @@
+import styles from "./NFText.module.css";
+
+import Image from "next/image";
```

Comment:
we can't use It becouse we build and use static and this is server side feature

Reply:
try optimize images with this library https://github.com/cyrilwanner/next-optimized-images or just use raw html img tag